tuya.cpp

```diff
@@ -172,19 +176,90 @@ void DeRestPluginPrivate::handleTuyaClusterIndication(const deCONZ::ApsDataIndic
         DBG_Printf(DBG_INFO, "Tuya debug 4 : Address 0x%016llX Payload %s\n" , ind.srcAddress().ext(), qPrintable(zclFrame.payload().toHex()));
         DBG_Printf(DBG_INFO, "Tuya debug 5 : Status: %d Transid: %d Dp: %d (0x%02X,0x%02X) Fn: %d Data %ld\n", status , transid , dp , dp_type, dp_identifier, fn , data);
         
-        //Sensor and light use same cluster, so need to make a choice for device that have both
-        if ((sensorNode) && (lightNode))
+        if (length > 4) //schedule command
+        {
+            
+            // Monday = 64, Tuesday = 32, Wednesday = 16, Thursday = 8, Friday = 4, Saturday = 2, Sunday = 1
+            // If you want your schedule to run only on workdays, the value would be W124. (64+32+16+8+4 = 124)
+            // The API specifies 3 numbers, so a schedule that runs on Monday would be W064.
+            // Workday = W124
+            // Not working day = W003
+            // Saturday = W002
+            // Sunday = W001
+            // All days = W127
+            
+            QString transitions = QString("");
```

Comment:
> `QString transitions = QString("");`

`QString transitions;`   is enough, it already initializes as empty string.

Reply:
I have see others one, in other code part (that was not from this PR), do I correct them too in same time ?

Reply:
It doesn't do harm, maybe for now just use the simpler version for new code.

Reply:
ok done only for tuya code.